src/crud.js

```diff
@@ -33,32 +33,35 @@ models: {
 
 export default (server, model, { prefix, defaultConfig: config, models: permissions }) => {
   const modelName = model._singular;
+  const permissionsIsArray = Array.isArray(permissions);
```

Comment:
Up to you, but I kind like the errors. It lets the user know that they passed an invalid config. Eventually, it might be a good idea to use Joi to validate the options though?